tsconfig.json

```diff
@@ -3,7 +3,7 @@
     "esModuleInterop": true,
     "exactOptionalPropertyTypes": true,
     "forceConsistentCasingInFileNames": true,
-    "lib": ["ES2020"],
+    "lib": ["ES2020", "dom"],
```

Comment:
Curious why we suddenly need this?

Reply:
Copied this from `utils` but doesn't seem necessary.